handbook/fasse.md

```diff
@@ -194,40 +194,51 @@ data/
 
 ## Step 8. Scratch Space
 
-Scratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. The high-performance storage and resource isolation capabilities of a scratch space enhance performance, streamline workflows, and protect valuable data. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
+A acratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
 
-We have created a scratch space within our FASSE space: 
+**When to use a scratch space?** Whenever you need a temporary storage for intermediate data during processing or computations, use it to store data that is being actively manipulated or processed but is not needed for long-term storage.
 
+The lab has scratch spaces available within FASSE and CANNON: 
+
+FASSE path: 
+```
+/n/holyscratch01/LABS/dominici_nsaph/Lab 
 ```
-Directory: /n/holyscratch01/LABS/dominici_nsaph/Lab 
+
+CANNON path:
 ```
-You should create your own directory in this scratch space based on your project name. For example: `/heat-stress-project`
+/n/holyscratch01/dominici_lab/Lab/
+```
+
+**How to use the scratch folder?** Create your own directory in this scratch space using your project name. For example, if your FASSE project name is `heat-stress-project`, then create a folder within the FASSE scratch folder: 
 
-Here is an instructions on how to use the scratch folder using R. Ideally a symlink the path folder should be incorporated within the project repo that is using the scratch space and not full paths.
+`mkdir /n/holyscratch01/LABS/dominici_nsaph/Lab/heat-stress-project`
 
-This is the example of your project path: 
+Instead of storing large files in your local project folder, write temporary files in this path.
+
+**Use a symlink to a scratch folder** Ideally, you should not include full paths in a repository's code. To avoid including full paths in your code, create a symlink to the scratch folder inside the `data/intermediate/` folder of your project using:
 
 ```
-project-name
-├── README.md
-├── data/
-├── code/
-├── figures/
-├── reports/
-├── results/
-└── .gitignore
+ls -s /n/holyscratch01/LABS/dominici_nsaph/Lab/<project_name> .
```

Comment:
do you need to navigate to the data/intermediate folder first before running the above line in terminal? the steps to accomplish this may need to be broken down more for the novice user

Reply:
I simplified it to /data instead because not all project structures has data/intermediate and added some description about /data.